Assets/Scripts/optionTransitionControl.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
```

Comment:
This Script in general seems to copy a lot of code from the menuTransitionControl is there maybe a way to have the code in only one script?

Reply:
Not from the way things are currently structured. I want each menu to be able to move independently of the other. Having two scripts means the code needed is simpler.

Reply:
Ok